src/app/zap-templates/zcl/zcl.json

```diff
@@ -273,7 +273,15 @@
         "Channel": ["Lineup", "CurrentChannel"],
         "Media Playback": ["SampledPosition"],
         "Application Launcher": ["CurrentApp"],
-        "Application Basic": ["Application"]
+        "Application Basic": ["Application"],
+        "Time Synchronization": [
```

Comment:
I'm less familiar with how this file relates to the zap generation, but why are these attributes defined here and not the others?

Reply:
This file lists the attributes that are always implemented via AttributeAccessInterface and hence should be forced to "External" storage in ZAP to not allocate (useless) memory to them.

This is automatic for lists, so they don't have to be listed here.

This is required for structs, because for those you can't allocate space in the store anyway, and ZAP can't usefully detect structs at runtime right now, long story.

For other attributes it depends on how they are implemented.

Reply:
Thanks.

Is DefaultNTP going through the AttributeAccessInterface? Or is that also a special case because it's a string?

Reply:
> Is DefaultNTP going through the AttributeAccessInterface?

That's impossible to tell from this PR, until the implementation happens....

Ideally, this PR would just add the structs to the json here, and the other bits would get added when they are implemented.

Reply:
I don't think we need to change anything here for now, right? I'm going to resolve this conversation and we could address changes in the implementation PR.

Reply:
Yeah, i think just be careful that the impl matches what's here, or change it in the impl pr